### ISSUE TITLE
Focus of the attention mask

### ISSUE THREAD

**Issue by aezco** [tengshaofeng/ResidualAttentionNetwork-pytorch#24]
I have question about the the soft attention mask. I have implemented residual attention blocks for specific domain (faces). How does the attention mask focus on specific regions of the face? such as forehead and so on???
